cmd/analyzer/expected_out/acl_testing3_detailed_explain.txt

```diff
@@ -0,0 +1,39 @@
+Explaining connectivity from 10.240.10.4 to vsi2-ky within test-vpc1-ky
```

Comment:
I assume this is a renaming of an old test and it was verified that the results are identical to the new file?

Reply:
yes